src/transformation/builtins/function.ts

```diff
@@ -32,7 +32,7 @@ export function transformFunctionPrototypeCall(
             return transformLuaLibFunction(context, LuaLibFeature.FunctionBind, node, caller, ...params);
         case "call":
             return lua.createCallExpression(caller, params, node);
-        default:
+        case "toString":
```

Comment:
This seems like a mistake, this should be default to catch everything we have not handled

Reply:
Since functions can have any properties, we don't want to throw errors on them being used. So I changed this to only error on JS lib functions that we don't handle.